packages/spacecat-shared-rum-api-client/test/fixtures/trafficSources.json

```diff
@@ -3,18 +3,36 @@
     "url": "https://www.aem.live/home",
     "total": 2620,
     "earned": 400,
-    "owned": 2220,
-    "paid": 0,
+    "owned": 2120,
+    "paid": 100,
     "sources": [
       {
         "type": "owned:direct",
-        "views": 2220
+        "views": 2120
       },
       {
         "type": "earned:search",
         "views": 400
+      },
+      {
+        "type": "paid:social",
+        "views": 100
       }
-    ]
+    ],
+    "vendors": {
```

Comment:
do we need this vendors object?

Reply:
removed this as this will be part of `sources` now, but `vendors` will still be there in the `getCTRByUrlAndVendor`